cpp/src/arrow/adapters/orc/adapter_test.cc

```diff
@@ -239,7 +241,9 @@ void AssertTableWriteReadEqual(const std::shared_ptr<Table>& input_table,
   write_options.row_index_stride = 5000;
   EXPECT_OK_AND_ASSIGN(auto writer, adapters::orc::ORCFileWriter::Open(
                                         buffer_output_stream.get(), write_options));
-  ARROW_EXPECT_OK(writer->Write(*input_table));
+  for (int iWrite = 0; iWrite < nb_writes; iWrite++) {
```

Comment:
We take the convention from the Google Style Guide that variable names should be lower case.

```suggestion
  for (int i_write = 0; i_write < nb_writes; i_write++) {
```